src/Frontend.elm

```diff
@@ -1180,9 +1318,9 @@ ticketInfo =
 # Tickets
 
 There is a mix of room types — singles, doubles, dorm style rooms
-suitable for up to four people. We will have to self-organize
-to distribute ourselves among the rooms and share bathrooms.
-The facilities for those who wish to bring a tent and camp
+suitable for up to four people. Attendees will self-organize
```

Comment:
(or 7 if you don't mind close quarters) Not sure how to word this to encourage if folks don' mind bunking up. It'd be great to have as many people on site as possible.